src/main/resources/swagger/api-docs.yaml

```diff
@@ -211,6 +211,43 @@ paths:
           description: Workspace does not exist
         500:
           description: Internal Server Error
+  /workspaces/{workspaceNamespace}/{workspaceName}/entities/copy:
+    post:
+      tags:
+      - Entities
+      operationId: copyEntities
+      summary: |
+        Copy entities from one workspace to another
+      produces:
+        - application/json
+      parameters:
+        - name: workspaceNamespace
+          description: Workspace Namespace
+          required: true
+          type: string
+          in: path
+        - name: workspaceName
+          description: Workspace Name
+          required: true
+          type: string
+          in: path
+        - name: body
```

Comment:
I was hesitant about going that route. The endpoint of /workspaces/entities/copy is the same as /workspaces/{workspacenamespace}/{workspacename} and there is potential for url collision. I chose this path more for consistency (and to avoid potential collisions) rather than efficiency.